plugin/src/main.ts

```diff
@@ -0,0 +1,85 @@
+import Ajv from "ajv";
+import { SchemaValidationLog, SchemaValidationResult } from "./pdk";
+import YAML from "js-yaml";
+import { parseYaml, XtpSchema, NormalizeError } from "@dylibso/xtp-bindgen";
+import JSON_SCHEMA from "../schema.json"
+
+/**
+ * Retrieves the JSON schema used for validation
+ *
+ * @returns {string} The JSON schema for xtp schema
+ */
+export function get_json_schemaImpl(): string {
+  return JSON.stringify(JSON_SCHEMA);
+}
+
+/**
+ * Checks if the input schema has any imports
```

Comment:
deciding if we need this or not

Reply:
@mhmd-azeez you'd be best to say. where is this used exactly?

Reply:
I think @nilslice was using this somewhere

Reply:
as far as i can see, this is the only place that uses it:
https://github.com/dylibso/xtp/blob/main/service/api/src/routes/api/v1/extension-points/index.ts#L109

Reply:
I think this is the only place we use this:

https://github.com/dylibso/xtp/blob/cf5cc59bc12750f9d6731b2f23276f044880d428/service/api/src/routes/api/v1/extension-points/index.ts#L109

We can probably replace it with a call to `validate_schema`